### DISCUSSION TITLE
Assistance Required for Adding Seat and Sending Invitation URL

### DISCUSSION THREAD

**Discussion by Intuz-GitHub** [community/community#133626] [Enterprise]
### Select Topic Area

General

### Body

Hello GitHub Support Team,

I am experiencing issues with adding a seat and sending an invitation URL from my account within our organization. Could you please assist in resolving these issues?

I followed all the steps to add a new user, but it's not working now.

I appreciate your help.

Best regards,
Utsav Dave

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @Intuz-GitHub !

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the `Enterprise` category. I've gone ahead and moved it for you. Good luck!
